src/assets/tests/digitech.json

```diff
@@ -0,0 +1,623 @@
+[
+  {
+    "name": "Wie nennt sich die für variable Codelänge gängige Codierung, bei der ein Baum gebildet wird?",
+    "type": "radio",
+    "answer": [
```

Comment:
it should be "answers" (take a look at the README example)
please fix this for any question

Reply:
my bad, thank you for advice